src/styles/index.css

```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  padding: 32px 19px 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 150%;
+  color: #060b35;
+  padding-bottom: 4px;
+  padding-left: 16px;
+  padding-right: 16px;
+  max-width: 198px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 140%;
+  color: #616070;
+  padding-bottom: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+  line-height: 112%;
+  padding-bottom: 24px;
+  padding-left: 16px;
```

Comment:
Repeated padding-left and padding-right values in .card__title, .card__code, and .card__price suggest that these could be refactored into a common class to avoid duplication.